Snowflakes for admin panel

Slimi1 opened the pull request:
This will add snowflakes to the admin panel aswell, works with custom themes (tested with slate). 


**Preview**:
![image](https://github.com/prplwtf/blueprint-snowflakes/assets/135278375/4bca226a-f610-4944-b04f-ba16a30301a9)

Slimi1 committed:
Update conf.yml

Update conf.yml to use the admin wrapper for admin panel

Slimi1 committed:
Create admin.wrapper.blade.php

Admin wrapper blade for snowflakes in admin panel

prplwtf reviewed (changes requested):
Could you rename `admin.wrapper.blade.php` to `wrapper.blade.php`?

Slimi1 committed:
Rename admin.wrapper.blade.php to wrapper.blade.php

Slimi1 committed:
Update conf.yml

Conf.yml now uses the file "wrapper.blade.php" instead of "admin.wrapper.blade.php"

Slimi1 reviewed (commented):
I have renamed the file from 
>  "admin.wrapper.blade.php" to > "wrapper.blade.php"

prplwtf commented:
Alright, thank you.

I'll wait a bit before merging this while I get the admin options ready.

Slimi1 commented:
Sounds good!

I've also updated the conf.yml to use the correct file
